mlir/test/Conversion/VectorToLLVM/vector-to-llvm.mlir

```diff
@@ -3042,13 +3042,13 @@ func.func @vector_store_index_scalable(%memref : memref<200x100xindex>, %i : ind
 
 // -----
 
-func.func @vector_store_op_0d(%memref : memref<200x100xf32>, %i : index, %j : index) {
+func.func @vector_store_0d(%memref : memref<200x100xf32>, %i : index, %j : index) {
```

Comment:
NIT or part of another MR, this test has args like %memref whereas other seem to follow argX type.

Reply:
Thanks for pointing this out!

There's actually more inconsistencies in this file, e.g. 
* `@vector_store_0d` vs `@scatter_index` (i.e. `@vector_{op_name}_{variant}` vs `@{op_name}_{variant}`.  

I was planning to send a dedicated refactor patch at the end :)